src/software/SfM/main_HyperSfM.cpp

```diff
@@ -115,25 +145,29 @@ int main(int argc, char **argv)
   tracks::TracksBuilder tracksBuilder;
   // List of features matches for each couple of images
   const openMVG::matching::PairWiseMatches & map_Matches = matches_provider->pairWise_matches_;
-  std::cout << "\n" << "Track building" << std::endl;
+  std::cout << "\n" << "...Track building" << std::endl;
 
   tracksBuilder.Build(map_Matches);
-  std::cout << "\n" << "Track filtering" << std::endl;
+  std::cout << "\n" << "...Track filtering" << std::endl;
   tracksBuilder.Filter();
-  std::cout << "\n" << "Track export to internal struct" << std::endl;
+  std::cout << "\n" << "...Track export to internal struct" << std::endl;
 
   //-- Build tracks with STL compliant type :
   tracksBuilder.ExportToSTL(map_tracks);
 
+  // choose which threshold to use
+  std::unique_ptr<SubmapThresholdChecker> threshold_checker = getThresholdChecker(tracks_threshold_clustering, view_threshold_clustering);
```

Comment:
Does it can be const?

Reply:
not if I want to std::move it a few lines below...But maybe this should be designed differently I'll think about it.